setup.cfg

```diff
@@ -48,7 +48,7 @@ VCS = git
 style = pep440
 versionfile_source = openff/interchange/_version.py
 versionfile_build = openff/interchange/_version.py
-tag_prefix = ''
+tag_prefix = v
```

Comment:
Not having this was the cause of my problems

Reply:
I think, without fully demonstrating, that this is necessary when some tags don't start with `v` but the recent ones do